full-stack-tests/native-app/src/backend/backend.ts

```diff
@@ -3,22 +3,22 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 
-// Sets up certa to allow a method on the frontend to get an access token
+// required to get certa to read the .env file - should be reworked
 import "@bentley/oidc-signin-tool/lib/certa/certaBackend";
```

Comment:
@wgoehrig this is necessary to get [this call](https://github.com/imodeljs/imodeljs/blob/3fd0ef4be53bbb4ba726d8ed56052b693b1df55e/tools/oidc-signin-tool/src/certa/certaBackend.ts#L21) to be hit. Doesn't seem right to me.